src/main/java/com/rackspace/feeds/feedscatalog/ResolveHostFilter.java

```diff
@@ -0,0 +1,94 @@
+package com.rackspace.feeds.feedscatalog;
+
+import com.rackspace.feeds.filter.TransformerUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.URL;
+import java.util.*;
+
+public class ResolveHostFilter implements Filter {
```

Comment:
Class comment added.

The xslt resolve the url of the href (by resolving the vip and tenantId) using the feedscatalog-resolve-host.xsl file.  Should I rename the class name to ResolveHrefFilter?